### DISCUSSION TITLE
OpenSource

### DISCUSSION THREAD

**Discussion by CodeTorso** [OpenCodeEra/.github#28] [General]
The true spirit of open-source

**Comment by CodeTorso** [comment]
Well well well,

I started doing open source in sept 2023. 

Open source was a new field for me.

When I joined, I learned a lot. This has been some of my best programming experiences.

This community was also a part of that.

I joined it like as a member at the start of Hacktober Fest. But I was in this way before that.

I contributed the most in Open Code Era homepage. 

Now, coming back to this community I don't feel that excitement. That drill within me. 

Like that of september. Many come to open source through Hacktober Fest, but they just leave after that.

Now I have learnt that

Open source is about contributing to problems that genuinely existed, not just building dumb frontend clones of websites that already exist. and that is the spirit of opensource. 

Contributing to big open source organisation codebases. Building communities. Or even having fun sharing leetcode problems or full-stack building projects with full collaboration as team. that should be the approach

I am sorry @AdarshRawat1 (or @AKACHI-4 ). I am leaving this community as I feel this community just does not fit with my beliefs of open-source. It is a also because I am kid about a 1000km away from the entire community which is not active on social media. I am yet to go to college.

Yeah learning Next, devops. see yeah.
<img width="832" alt="image" src="https://github.com/OpenCodeEra/.github/assets/129188028/56f6c13f-3cb7-4893-bafb-a3862d5cad07">

**Comment by AKACHI-4** [comment]
Hey @CodeTorso ,

Thanks for sharing your open-source journey. It's clear you've learned a lot and grown. Your focus on solving real problems aligns with the spirit of open source.

It's okay to move on from a community that no longer resonates with you. Best of luck with Next and DevOps! Feel free to return anytime.

Take care,

**Comment by AdarshRawat1** [comment]
@CodeTorso  , your contributions to this journey have been invaluable, and it has been a pleasure interacting with you. I wish you all the best in your future endeavours.

### Quick Note - 
Please understand, the design of Open Code Era is specifically to help budding contributors to get comfortable with open-source community, as real-world projects can often be overwhelming for newcomers. 

> Open source is about contributing to problems that genuinely existed, not just building dumb frontend clones of websites that already exist. and that is the spirit of opensource.

Our goal is to provide a foundation, helping members grasp the basics of Git, GitHub, and PR etiquettes, while also giving them a taste of the open-source community. (Front end being the easiest to learn and start contributing.) 

_Once these foundations have been built, it's natural and encouraged for you to move on to more complex projects that align with your professional growth and open-source spirit._ 

I appreciate the time and effort you've invested in Open Code Era. I hope that your open-source journey continues to be fruitful and inspiring. Good luck with learning Next and DevOps. See you around.
